src/test/java/teammates/test/cases/action/InstructorCourseRemindActionTest.java

```diff
@@ -120,6 +138,7 @@ public void testExecuteAndPostProcess() throws Exception {
                 Const.ParamsNames.COURSE_ID, courseId
         };
         remindAction = getAction(addUserIdToParams(instructorId, submissionParams));
+        remindAction.setCurrentUrl(Const.ActionURIs.INSTRUCTOR_COURSE_DETAILS_PAGE);
```

Comment:
This line and line 178 are not necessary. These test cases refer to the remind all option from INSTRUCTOR_COURSE_DETAILS_PAGE I believe, so there's no need to explicitly set the url.